src/main/java/me/coley/recaf/config/ConfBackend.java

```diff
@@ -38,6 +38,11 @@
 	 */
 	@Conf("backend.recentsave")
 	public String recentSaveWorkspace = CURRENT_DIR;
+	/**
+	 * Recent path used by the save-map dialog.
+	 */
+	@Conf("backend.recentsave")
```

Comment:
The config identifier should be different here so it doesn't conflict with the workspace one above

Reply:
I thought this was intentional as `recentSaveApp` and `recentSaveWorkspace` already use the same identifier.
I'll change all 3, guessing it was an accident all along. Feel free to request more changes if that isn't what you want.

Reply:
Oh crap. Looks like I've missed something all along.